webapp/src/components/modals/attach_comment_to_issue/index.js

```diff
@@ -0,0 +1,25 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {getPost} from 'mattermost-redux/selectors/entities/posts';
+
+import {closeAttachCommentToIssueModal, attachCommentToIssue} from 'actions';
+
+import AttachCommentToIssue from './attach_comment_to_issue';
+
+const mapStateToProps = (state) => {
+    const postId = state['plugins-github'].attachCommentToIssueModalForPostId;
```

Comment:
Can we import the plugin id, rather than hardcoding here?

Reply:
done